priv/perf/apps/load_test/lib/service/datadog/api.ex

```diff
@@ -131,26 +145,16 @@ defmodule LoadTest.Service.Datadog.API do
       {:ok, %{body: body}} ->
         Logger.error("failed to resolve monitors #{inspect(body)}")
 
-        case retries do
-          0 ->
-            {:error, body}
+        Process.sleep(1_000)
 
-          _ ->
-            Process.sleep(1_000)
-            do_resolve_monitors(params, retries - 1)
-        end
+        do_resolve_monitors(params, retries - 1)
 
-      {:error, error} = other ->
+      {:error, error} ->
         Logger.error("failed to resolve monitors #{inspect(error)}")
```

Comment:
nitpick: I now is not part of this PR change. But might consider using warn level and add some words to indicate it is going to retry as it is still fighting to get out of the error.